_posts/2017-12-11-away-week-prague/index.md

```diff
@@ -0,0 +1,62 @@
+
+One core reason we’re able to be a valuable resource for our users is our understanding of the challenges they’re facing, and we’ve taken steps to more comprehensively understand the process and benefits of A/B testing. Knowing that A/B testing is a high priority for many of our users, and having come across positive feedback for [Wasabi][wasabi], an open source testing platform by Intuit, one of our hackathon teams focused on setting up and integrating a Wasabi instance, allowing us to configure a wide range of experiments for our website and products. Given the success of this test, we’re even exploring offering Wasabi as a tool for our users, letting them quickly and seamlessly test hypotheses about their website or product and collect data with their Snowplow pipeline.
+
+The Snowplow user experience was top of mind for our hackathon teams. Beyond A/B testing, we also had a team focused on data modeling. The data modeling step of the digital analytic process can often be the most challenging, with access to so much data often making it difficult to know what questions to ask to get started. This team explored ways we can improve the data modeling process for our users, specifically focusing on three different areas: building a DSL, porting data to graph databases using [Neo4j][neo], and making the modeling process more accessible with PySpark. You can look forward to further information about a more streamlined, user friendly data modeling process in a future post. To further empower our users, another team set about creating what they called Snowplow Wire. Using Wire, we turned our analytic capabilities inward on our pipeline, allowing us to collect brand new data on the health and performance of the pipelines we manage as well as how to socialize this metadata to our users so they can benefit from it as well.
+
+![team-hackathon][hackathon]
+
+<h2 id="today here tomorrow the world">Today here, tomorrow the world</h2>
+
+While looking towards the future of the Snowplow product offerings, we took the opportunity of having our team together in one location to do some creative problem solving the the form of war gaming as many potential security breaches we could imagine, from the mundane to the wildly outlandish. While our Tech Ops team has already guarded us against the majority of these scenarios, as an exercise it helped us identify and plan for potential challenges we would have otherwise not considered. The integrity of our users’ data is our number one priority, and time spent planning for any possible threat scenario is time well spent.
+
```

Comment:
Let's make the point more explicit: data security is a something that our whole team at Snowplow are concerned with. The away week provided an opportunity to get people across our different teams to war game together with a view to identifying new vulnerabilities and approaches for guarding against them.